core/domain/suggestion_jobs_one_off.py

```diff
@@ -215,8 +215,9 @@ def map(item):
         if suggestion.suggestion_type == (
                 suggestion_models.SUGGESTION_TYPE_ADD_QUESTION):
             # Set the language code to be the language of the question.
-            suggestion.language_code = suggestion.change.question_dict[
-                'language_code']
+            suggestion.change.question_dict['language_code'] = (
+                constants.DEFAULT_LANGUAGE_CODE)
```

Comment:
@seanlip, Should I create a separate job for only populating the question suggestion models which have None language code? OR it's fine to change here and rerun this existing `PopulateSuggestionLanguageCodeMigrationOneOffJob` job?

[See PR description to know the reason for these changes.]

Reply:
I think here is OK, but either is fine!

Reply:
I was thinking of this fix too @DubeySandeep - of forcing the question suggestion language code to be the default instead of grabbing it from the question_dict

Reply:
> this means that some question suggestions had None in their question_dict and others had the default though right?

@kaylahardie Yeah, some of the older models had None language code.

Reply:
I see - well thanks @DubeySandeep for making this PR :)